site/en/docs/extensions/whatsnew/index.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: 'layouts/doc-post.njk'
+
+title: What's new in Chrome extensions
+
+# This appears below the title and is an optional teaser
+# subhead: 
+
+# This appears in the ToC of the project landing page at
+# /docs/[project-name]/. It also appears in the <meta description> used in 
+# Google Search.
+description: 'Recent changes to the Chrome extensions platform, documentation, and policy'
+
+# The publish date
+date: 2021-02-25
+
+# An optional updated date
+# updated: 2020-10-16
+
+# A list of authors. These usernames correspond to the keys in the
+# _data/authorsData.json file.
+
+# Note: disabling the linter for duplicate headings because this isn't hierarchical and it needs
+# smaller font headings.
+
+---
+<!--lint disable no-duplicate-headings-->
+<!--lint disable first-heading-level-->
+
+Check this page often to learn about changes to the Chrome extensions platform,
+its documentation, and related policy or other changes.
+
+### 2021.01.19: Manifest V3 launched
+
+With the launch of Chrome 88, the extensions platform now supports extensions
```

Comment:
```suggestion
With the release of Chrome 88, the extensions platform now supports extensions
```

(Or even just "As of Chrome 88")

I don't know why, but I feel like I haven't heard "launch" applied to Chrome releases frequently.